lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/dllmain.cpp

```diff
@@ -1244,6 +1257,37 @@ void eos_create(EOSConfig& eosConfig)
         steam_platform.SteamMajorVersion = eos_steam_config.steamSDKMajorVersion;
         steam_platform.SteamMinorVersion = eos_steam_config.steamSDKMinorVersion;
 
+        // For each element in the array (each of which is a string of an api version information)
+        // iterate across each character, and at the end of a string add a null terminator \0
+        // then add one more null terminator at the end of the array
+        std::vector<char> steamApiInterfaceVersionsAsCharArray = std::vector<char>();
```

Comment:
The assignment here is not needed.

Reply:
You're right. I had assumed that I needed to set it to something before I can start managing it, but it appears as though `std::vector<>` lets you use it right away without the assignment. Learned something new about C++!